frontend/src/components/emissionResults/emissionResults.js

```diff
@@ -1,25 +1,65 @@
-import React, { useState } from "react";
-
 const EmissionResults = ({ emissions, renderEmissions }) => {
   if (renderEmissions === true) {
     return (
       <>
-        <div id="emissionResults" className="mt-24">
+        <div id="emissionResults" className="mt-2">
           <div>
-            <div className="stat shadow">
+            <div className="stats stats-vertical shadow">
               <div className="stat">
-                <div className="stat-title">CO2e Air</div>
+                <div className="stat-title">CO2e by Air</div>
```

Comment:
You could maybe create these with a forEach loop, since it's a bit repetitive creating each sticker individually.